src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -1148,6 +1169,110 @@ private static boolean shouldOmitClass(String classname) {
     }
   }
 
+  /**
+   * Returns classes in the given package. Does not include classes in sub-packages. Ignores
+   * non-accessible classes.
+   *
+   * @param packageName a package name
+   * @param accessibility the accessibility predicate
+   * @return classes in package {@code packageName}
+   */
+  private static Set<@ClassGetName String> getClassnamesFromPackage(
+      String packageName, AccessibilityPredicate accessibility) {
+    Set<@ClassGetName String> classnames = new TreeSet<>();
+
+    for (String path : Globals.getClassPath().split(File.pathSeparator)) {
+      File location = new File(path);
+      if (location.isFile() && location.getName().endsWith(".jar")) {
+        classnames.addAll(getClassesWithPackageFromJar(location, packageName, accessibility));
+      } else if (location.isDirectory()) {
+        classnames.addAll(getClassesWithPackageFromDirectory(location, packageName, accessibility));
+      }
+    }
+
+    return classnames;
+  }
+
+  /**
+   * Searches in the directory on the CLASSPATH for classes with the given package.
+   * Converts directory and package name to directory, where classes with proper
+   * package should be present. If it exists returns all found classes
+   * (subdirectories excluded), if not returns empty set.
+   *
+   * @param directory a directory on the CLASSPATH
+   * @param packageName a package name
+   * @param accessibility the accessibility predicate
+   * @return classes with the given package that were found in the specified directory
+   */
+  private static Set<@ClassGetName String> getClassesWithPackageFromDirectory(
+      File directory, String packageName, AccessibilityPredicate accessibility) {
+    String packageNameAsFile = packageName.replace(".", File.separator);
+    File packageDirectory = // to find needed directory so we don`t need to check every file
+      directory.toPath().resolve(packageNameAsFile).toFile();
+    if (packageDirectory.exists() && packageDirectory.isDirectory()) {
+      Set<@ClassGetName String> classnames = new TreeSet<>();
+      for (File file :
+          packageDirectory.listFiles(f -> f.isFile() && f.getName().endsWith(".class"))) {
+
+        String relativePath =
+            directory.toPath().relativize(file.toPath()).toString();
+        @ClassGetName String classname = Signatures.binaryNameToClassGetName(
```

Comment:
Every binary name is a classgetname (see the [type hierarchy](https://checkerframework.org/manual/#signature-annotations)), so the call to `binaryNameToClassGetName` is a no-op (except in erroneous circumstances).  You can remove it.  You also don't need the `@ClassGetName` annotation; it will be inferred.